dynamallow/model.py

```diff
@@ -221,7 +230,9 @@ def get(cls, consistent=False, **kwargs):
         :param bool consistent: If set to True the get will be a consistent read
         :param \*\*kwargs: You must supply your hash key, and range key if used, with the values to get
         """
-        return cls.new_from_raw(cls.Table.get(consistent=consistent, **kwargs))
+        value = cls.Table.get(consistent=consistent, **kwargs)
```

Comment:
Instead of checking if `.get` returned `None` here, as well as all of the `if raw is not None` in your comprehsions below, let's just move the check for `None` directly into `new_from_raw`.  Less repetition.